assets/layers/doctors/doctors.json

```diff
@@ -38,6 +38,16 @@
         "phone",
         "email",
         "website",
+        {
+            "question": {
+                "en": "What is the name of this doctors place?"
+            },
+            "render": "This doctors place is called {name}",
```

Comment:
The translation program won't pick this up, please use `{"en": "This doctors place is called {name}"}` instead.